AuxTel/AuxTel-Troubleshooting/ATCS-Troubleshooting/ATMount-fails-move-timeout.rst

```diff
@@ -0,0 +1,59 @@
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Y.-AlSayyad*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *P.-Venegas*, *K.-Aubel*, *K.-Pena*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-Mount-Fails-to-Move-and-Times-Out:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################################
+AuxTel Mount Fails to Move and Times Out
+#########################################
+
+.. _AuxTel-Mount-Fails-to-Move-and-Times-Out-Procedure-Overview:
+
+Overview
+========
+
+In this troubleshooting guide, we address an issue where the telescope times out while getting into position. 
+All ATCS CSCs indicated no faults. However, the ATMCS Engineering User Interface System (EUI) showed several low air pressure faults. An air leak was discovered at the telescope base. 
```

Comment:
One sentence per line. I'm not sure if the air leak comment is necessary.

Reply:
Agree